### DISCUSSION TITLE
Import Requirement Files & Constraint File

### DISCUSSION THREAD

**Discussion by benbenbang** [python-poetry/poetry#9449] [Show and tell]
Hello team,

I wanted to share a small project I built for importing **requirement txt to `TOML`**. I have reworked it to be part of the poetry plugin: [Poetry Import Plugin](https://github.com/benbenbang/poetry-import-plugin). 

Th project’s goals are to 

- import requirement.txt files to the given group you assign with one command. Note that to reduce the complexity, file is the minimum granularity. Meaning that you will need to fine grind the requirement files, since the plugin won’t parse the file line by line and read customized/non-standard markers.
- Constraint file: it can take a constraint file to work **almost** the same way like pip

Hope this can help with some requests for the import or constraint files.

Contribution is always welcome! Don’t hesitate to share your feedback with me.

Thank you

**Comment by Ricky587** [comment]
Hey, thanks for the plugin. It works perfectly!!

**Reply by benbenbang** [reply]
Thanks 🙏🏻 Good to know it helped you.
